src/state-summary/state-card-vacuum.html

```diff
@@ -0,0 +1,205 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+<link rel='import' href='../../bower_components/paper-dropdown-menu/paper-dropdown-menu.html'>
+<link rel='import' href='../../bower_components/paper-listbox/paper-listbox.html'>
+<link rel='import' href='../../bower_components/paper-item/paper-item.html'>
+<link rel='import' href='../../bower_components/paper-icon-button/paper-icon-button.html'>
+
+<link rel="import" href="../components/entity/state-info.html">
+<link rel="import" href="../components/entity/ha-entity-toggle.html">
+
+<dom-module id="state-card-vacuum">
+  <template>
+    <style is="custom-style" include="iron-flex iron-flex-alignment"></style>
+    <style>
+      :host {
+        @apply(--paper-font-body1);
+        line-height: 1.5;
+      }
+
+      .status-subtitle {
+        color: var(--secondary-text-color);
+      }
+
+      .container-extra-controls {
+        margin-top: 1.5em;
+      }
+
+      paper-item {
+        cursor: pointer;
+      }
+    </style>
+
+    <div class='horizontal justified layout'>
+      <state-info state-obj="[[stateObj]]" in-dialog='[[inDialog]]'></state-info>
+      <ha-entity-toggle state-obj="[[stateObj]]" hass='[[hass]]'></ha-entity-toggle>
+    </div>
+
+    <div hidden$='{{inDialog}}'>
```

Comment:
> These should all be removed from the state card and be moved to the more-info card

That was my first try, but I changed it because (for me) it looked better with the controls in top of the history chart, with the attrs table below, but I'll change it to use the more-info card (with the last change I wasn't using it)

> The state cards are all name + primary control/state. No extra lines are allowed.

I took the climate card as example (first time here), and thought that the status, fan speed and battery could be in a line below. So this is not allowed?

<img width="352" alt="captura de pantalla 2017-07-29 a las 9 29 12" src="https://user-images.githubusercontent.com/9748301/28743059-8929b18a-7440-11e7-911b-695b93d9139d.png">

If so, I'll remove it, but I think I'll use my first attempt as custom_UI, I like the idea of seeing more than a switch here.